ui/nuxeo-button-styles.js

```diff
@@ -69,6 +69,22 @@ const template = html`
         paper-button iron-icon {
           @apply --nx-button-icon;
         }
+
+        paper-button.horizontal-button iron-icon + span {
```

Comment:
nit: wondering if we could ditch the "button" part of the class because it seems redundant when putting it together with `paper-button`. WDYT?
```suggestion
        paper-button.horizontal iron-icon + span {
```

Reply:
Not really, because that naming would collapse with the ones used by polymer for layouts. I am open to other suggestions.